File: Ent_UsAdmin/CrearNotificacion.vb
Imports System.ComponentModel
Imports MySql.Data.MySqlClient

Public Class CrearNotificacion
    Dim Creada As Boolean
    Dim ncargando As Cargando
    Private Sub btnSesiones_Click(sender As Object, e As EventArgs) Handles btnSesiones.Click
        BuscarSesion.Show()

    End Sub

    Private Sub Button1_Click(sender As Object, e As EventArgs) Handles Button1.Click
        ncargando = New Cargando
        ncargando.Show()
        ncargando.MonoFlat_Label1.Text = "Creando Notificación"
        ncargando.TopMost = True
        Button1.Enabled = False
        BackgroundNotificacion.RunWorkerAsync()

    End Sub

    Private Sub BackgroundNotificacion_DoWork(sender As Object, e As System.ComponentModel.DoWorkEventArgs) Handles BackgroundNotificacion.DoWork
        Try
            Dim conexionLogin As MySqlConnection
            conexionLogin = New MySqlConnection()
            conexionLogin.ConnectionString = "server=www.prestamosconfia.com;user id=ajas;pwd=123456;port=3306;database=USRS"
            conexionLogin.Open()
            Dim comandoLogin As MySqlCommand
            Dim consultaLogin As String
            consultaLogin = "insert into Notificaciones values(null,'" & ComboTipo.Text & "',0,'" & nmusr & "','" & txtUsuario.Text & "','','" & txtMensaje.Text & "','',0,'" & txtIdSesion.Text & "','" & Date.Now.ToString("yyyy-MM-dd") & "','" & Date.Now.ToString("HH:mm:ss") & "','0','','','','',''); "
            comandoLogin = New MySqlCommand
            comandoLogin.Connection = conexionLogin
            comandoLogin.CommandText = consultaLogin
            comandoLogin.ExecuteNonQuery()
            conexionLogin.Close()
            Creada = True
        Catch ex As MySqlException
            Creada = False
            MessageBox.Show(ex.Message)
        End Try

    End Sub

    Private Sub BackgroundNotificacion_RunWorkerCompleted(sender As Object, e As RunWorkerCompletedEventArgs) Handles BackgroundNotificacion.RunWorkerCompleted
        If Creada Then
            MessageBox.Show("Notificación creada con éxito")
        Else
            MessageBox.Show("Ha ocurrido un error, inténtelo de nuevo")
        End If
        Button1.Enabled = True
        ncargando.Close()
    End Sub
End Class

File: Ent_UsAdmin/Notificaciones/CrearNotificacion.cs
using System;
using System.ComponentModel;
using System.Windows.Forms;
using MySql.Data.MySqlClient;

namespace ConfiaAdmin
{

    public partial class CrearNotificacion
    {
        private bool Creada;
        private Cargando ncargando;

        public CrearNotificacion()
        {
            InitializeComponent();
        }
        private void btnSesiones_Click(object sender, EventArgs e)
        {
            My.MyProject.Forms.BuscarSesion.Show();

        }

        private void Button1_Click(object sender, EventArgs e)
        {
            ncargando = new Cargando();
            ncargando.Show();
            ncargando.MonoFlat_Label1.Text = "Creando Notificación";
            ncargando.TopMost = true;
            Button1.Enabled = false;
            BackgroundNotificacion.RunWorkerAsync();

        }

        private void BackgroundNotificacion_DoWork(object sender, DoWorkEventArgs e)
        {
            try
            {
                MySqlConnection conexionLogin;
                conexionLogin = new MySqlConnection();
                conexionLogin.ConnectionString = "server=www.prestamosconfia.com;user id=ajas;pwd=123456;port=3306;database=USRS";
                conexionLogin.Open();
                MySqlCommand comandoLogin;
                string consultaLogin;
                consultaLogin = "insert into Notificaciones values(null,'" + ComboTipo.Text + "',0,'" + Module1.nmusr + "','" + txtUsuario.Text + "','','" + txtMensaje.Text + "','',0,'" + txtIdSesion.Text + "','" + DateTime.Now.ToString("yyyy-MM-dd") + "','" + DateTime.Now.ToString("HH:mm:ss") + "','0','','','','',''); ";
                comandoLogin = new MySqlCommand();
                comandoLogin.Connection = conexionLogin;
                comandoLogin.CommandText = consultaLogin;
                comandoLogin.ExecuteNonQuery();
                conexionLogin.Close();
                Creada = true;
            }
            catch (MySqlException ex)
            {
                Creada = false;
                MessageBox.Show(ex.Message);
            }

        }

        private void BackgroundNotificacion_RunWorkerCompleted(object sender, RunWorkerCompletedEventArgs e)
        {
            if (Creada)
            {
                MessageBox.Show("Notificación creada con éxito");
            }
            else
            {
                MessageBox.Show("Ha ocurrido un error, inténtelo de nuevo");
            }
            Button1.Enabled = true;
            ncargando.Close();
        }
    }
}